.github/workflows/build.yml

```diff
@@ -244,6 +244,7 @@ jobs:
           touch ${{ github.workspace }}/Data/DebugDatabaseDumps/keep
 
           ./flow package:list --loading-order
+          FLOW_CONTEXT=Testing/Behat ./flow doctrine:migrate --quiet
```

Comment:
(how) is that related?

Reply:
The behat tests are not working without this change, as the CatchUpHook expects the doctrine:migrate command to be executed. But it's not when we run the `Neos.ContentRepository.BehavioralTests`.

This was not an issue before, as these tests where only relying on the projection tables. But the new CatchUpHook now, gets executed also in these test cases.

https://github.com/neos/neos-development-collection/blob/9.0/.composer.json#L34-L41

Reply:
That is our long standing: you have to do doctrine migrate sometimes problem otherwise it will not work problem that youll fix with the catchup thing as far as i know. see slack: https://neos-project.slack.com/archives/C04PYL8H3/p1727209924914109?thread_ts=1727117857.409749&cid=C04PYL8H3

Reply:
Upps i was mistaken, this is just really similar to this problem: https://github.com/neos/neos-development-collection/pull/5005

In https://github.com/neos/neos-development-collection/issues/4878 i also discovered that disabling the catchup hooks will speed up the tests ... and as they are just run along without any assertions in the core tests we should consider doing this some point.

Reply:
Yes, true. But we need to be able to enable it for the test cases where we test the catchup hook 🤷‍♂️